Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/CustomImageComparatorTest.cs

```diff
@@ -10,15 +10,19 @@ public class CustomImageComparatorTest : BaseTest
         [Test(Description = "TC-0005 Check Custom Image Comparator")]
         public void TC0005_CheckCustomImageComparator()
         {
-            CustomImageComparator.Init();
+            var firstImage = Image.FromFile(ResourceConstants.FirstTestImage);
 
-            var firstImage = Image.FromFile(PathConstants.FirstTestImage);
+            var firstImageWidth = firstImage.Width;
 
-            var secondImage = Image.FromFile(PathConstants.SecondTestImage);
+            var firstImageHeight = firstImage.Height;
 
-            var compareResult = CustomImageComparator.Compare(firstImage, secondImage);
+            var secondImage = Image.FromFile(ResourceConstants.SecondTestImage);
 
-            Assert.AreEqual(compareResult, 0, "The images should be the same");
+            var customImageComparator = new CustomImageComparator(0.1f, firstImageWidth, firstImageHeight);
```

Comment:
0.1f - magic number

We should refactor this test method to use some steps

Reply:
0.1f - the magic number - was updated. A new step has not been added. It is necessary to discuss.